### DISCUSSION TITLE
Lock Screen Wallpapers

### DISCUSSION THREAD

**Discussion by PavloMuravyov** [elementary/triage#514] [Q&A] [answered]
Hello. Are there any ways to change wallpaper on lock screen , instead of grey color?  Version  Elementary 7.1

**Comment by teamcons** [comment, marked as answer]
Hello

Lock screen is Lightdm, so tutorials to change background of lightdm should work
From a quick glance, the background is in a configuration file in /etc/lightdm or some subfolder of it.

You will need to open as root/administrator the file to change, and have a variable "background=" followed by the path to the background you want to set (ideally, within the same folder ? i am not sure if lightdm can access the files of an unlogged user, but i doubt it)
